### ISSUE TITLE
[Assistive tech and device support] Page functionality isn't keyboard accessible. (11.05.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#47164]
#### VFS team name
Digital Health Platform

#### VFS product name
Connect Your Device

#### VFS feature name
Device Connection Experience

#### Point of Contact/Reviewers
Brian DeConinck (@briandeconinck) - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Page functionality isn't keyboard accessible.

### Issue Details
When a Fitbit is connected, the Disconnect button is activated by pressing any key, rather than just [Enter] or [Spacebar]. This is unexpected behavior, and makes it impossible for keyboard-only users to reach the FAQs or any other interactive elements further down the page.

### Link, screenshot or steps to recreate
Steps to reproduce:

1. Log into staging and connect a device for the user.
2. Press [Tab] to navigate through the page content. A focus indicator will move to the Disconnect button to show that it has focus.
3. Press [Tab] again. Notice that this opens the "Disconnect device" modal.
4. Press [Tab] twice to move focus to "Cancel," and press [Enter] to cancel and close the modal.
5. Press [Tab] again. Focus once again returns to the Disconnect button.
6. Press any key on your keyboard. Notice that the Disconnect button is activated and the modal opens. (Except [Esc], [Caps Lock], and [Print Screen], which don't seem to be an issue!)

### VA.gov Experience Standard
[Category Number 11, Issue Number 05](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.1.1_A

---

### Platform Recommendation
I think there's some JS listener that's over-eager and should be fixed or removed. I'm also noticing a console error that may be related.

For reference, here's the expected behavior for keyboard navigation is that `<button>` elements are activated by [Enter] or [Spacebar]. (This should be baked into the `button` element, no JS required.)

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [x] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by robbie1226** [comment]
The reported issue was fixed on staging and passed QA within the team. I will close the ticket per the instructions under the "VFS Team Tasks to Complete" section.
